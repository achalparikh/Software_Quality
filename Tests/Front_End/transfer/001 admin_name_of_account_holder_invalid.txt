login
admin
lala llaa
logout
quit