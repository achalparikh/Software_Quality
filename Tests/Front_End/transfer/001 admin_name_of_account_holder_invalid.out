Enter user type - "standard" or "admin":
You are now logged in as an admin
COMMAND: transfer - enter user:
ERROR: User "lala llaa" does not exist. Try again
You are now logged out
